yarn.lock

```diff
@@ -0,0 +1,35 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Not sure we want a yarn.lock or package-json.lock in the PR, there are none in the original source code.